test_factorial (__main__.TestExercice) ... FAIL
test_prefixes (__main__.TestExercice) ... FAIL
test_summation (__main__.TestExercice) ... ok

======================================================================
FAIL: test_factorial (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 50, in test_factorial
    'Erreur dans le remplacement de caractère'
AssertionError: Lists differ: [0, 0, 0, 0] != [1, 6, 40320, 3628800]

First differing element 0:
0
1

- [0, 0, 0, 0]
+ [1, 6, 40320, 3628800] : Erreur dans le remplacement de caractère

======================================================================
FAIL: test_prefixes (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 20, in test_prefixes
    'Mauvaise identification de la parité de la longueur de la chaine'
AssertionError: Lists differ: ['Jack', 'Kack', 'Lack', 'Mack', 'Nack', 'Oack'] != ['Jack', 'Kack', 'Lack', 'Mack', 'Nack', 'Oack', 'Pack']

Second list contains 1 additional elements.
First extra element 6:
'Pack'

- ['Jack', 'Kack', 'Lack', 'Mack', 'Nack', 'Oack']
+ ['Jack', 'Kack', 'Lack', 'Mack', 'Nack', 'Oack', 'Pack']
?                                                ++++++++
 : Mauvaise identification de la parité de la longueur de la chaine

----------------------------------------------------------------------
Ran 3 tests in 0.003s

FAILED (failures=2)
